src/tables/stat.rs

```diff
@@ -0,0 +1,336 @@
+//! A [Style Attributes Table](https://docs.microsoft.com/en-us/typography/opentype/spec/stat) implementation.
+
+use crate::{
+    parser::{Offset, Offset16, Offset32, Stream},
+    Fixed, FromData, LazyArray16, Tag,
+};
+
+/// Axis-value pairing.
+#[derive(Clone, Copy, Debug)]
+pub struct AxisValue {
+    /// Zero-based index into [`Table::axes`].
+    pub axis_index: u16,
+    /// Numeric value for this axis.
+    pub value: Fixed,
+}
+
+impl FromData for AxisValue {
+    const SIZE: usize = 6;
+
+    fn parse(data: &[u8]) -> Option<Self> {
+        let mut s = Stream::new(data);
+        let axis_index = s.read::<u16>()?;
+        let value = s.read::<Fixed>()?;
+
+        Some(AxisValue { axis_index, value })
+    }
+}
+
+/// List of axis value tables.
+#[derive(Clone, Debug)]
+pub struct AxisValueTables<'a> {
```

Comment:
This should become a `Subtables` as well. The only table here should be the STAT itself.